server/@types/viewModels/index.d.ts

```diff
@@ -77,7 +77,7 @@ declare module 'viewModels' {
     updatedAt: Date
     updatedAtPrisonName?: string
     targetCompletionDate: Date
-    note?: string
+    notesByType: Record<'GOAL' | 'GOAL_ARCHIVAL' | 'GOAL_COMPLETION', Array<Note>>
```

Comment:
The single `note` field has been removed from the Goal view model type, and replaced with a `notesByType` structure